storage/database/db_migration.go

```diff
@@ -0,0 +1,120 @@
+// Modifications Copyright 2020 The klaytn Authors
```

Comment:
If you creates this code, you can use our copyright.

```
// Copyright 2019 The klaytn Authors
// This file is part of the klaytn library.
...
```

Reply:
review reflected :) Thanks